src/utils/PipeHelpers.ts

```diff
@@ -23,6 +23,8 @@ export const readableStream = (handler: () => Promise<any>): stream.Readable =>
       }
       if (value) {
```

Comment:
I don't know where exactly this util is used, but just to be sure, it's okay that _any_ falsy value will terminate the stream, including 0, '', etc.?